common/src/main/java/tachyon/Pair.java

```diff
@@ -74,8 +62,15 @@ public void setFirst(T1 first) {
   }
 
   /**
+   * @return the second element of the pair.
+   */
+  public T2 getSecond() {
```

Comment:
Generally we try to keep the methods separated by visibility and then alphabetical order, was this auto formatted?